common/libs/units.ts

```diff
@@ -103,7 +103,9 @@ const fromTokenBase = (value: TokenValue, decimal: number) =>
 const toTokenBase = (value: string, decimal: number) =>
   TokenValue(convertedToBaseUnit(value, decimal));
 
-const isEtherUnit = (unit: string) => unit === 'ether' || unit === 'ETH';
+const isNetworkUnit = (unit: string) =>
+  // TODO: find a better strategy for determining if unit is network unit
+  unit === 'ETH' || unit === 'ETC' || unit === 'UBQ' || unit === 'EXP';
```

Comment:
You should be able to derive the network unit by reducing the network config state

Reply:
Yep, unfortunately this wouldn't cover custom networks, so I think this'll need to take in the network object.